.github/workflows/publish-abac-nuget.yml

```diff
@@ -0,0 +1,36 @@
+name: Pack and publish Altinn.Authorization.ABAC nuget package
+
+on:
+  pull_request:
+    branches: infrastructure/publish-nuget-action
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    #if: startsWith(github.ref, 'refs/tags/Altinn.Authorization.ABAC-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            2.0.x
+      - name: Install deps
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet restore
+      - name: Build
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet build --configuration Release --no-restore -p:Deterministic=true -p:BuildNumber=${{ github.run_number }}
+      - name: Pack and publish Altinn.Authorization.ABAC
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet pack --configuration Release --no-restore --no-build -p:BuildNumber=${{ github.run_number }} -p:Deterministic=true
+          dotnet nuget push bin/Release/*.nupkg --source https://api.nuget.org/v3/index.json --api-key ${{ secrets.NUGET_API_KEY }}
```

Comment:
Don't interpolate secrets/variables as scripts. Rather pass them in as env variables.

See https://github.com/Altinn/altinn-authorization-utils/blob/a1deeb53af9573579aee66e0f6710691f2ac4c3f/.github/workflows/release.yml#L107